minecraft/dial.go

```diff
@@ -56,8 +56,9 @@ type Dialer struct {
 
 	// DownloadResourcePack is called individually for every texture and behaviour pack sent by the connection when
 	// using Dialer.Dial(), and can be used to stop the pack from being downloaded. The function is called with the UUID
-	// and version of the resource pack, and the boolean returned determines if the pack will be downloaded or not.
-	DownloadResourcePack func(id uuid.UUID, version string) bool
+	// and version of the resource pack, and the total amount of packs. The boolean returned determines if the pack will
+	// be downloaded or not.
+	DownloadResourcePack func(id uuid.UUID, version string, totalPacks int) bool
```

Comment:
Probably a good idea to also include the current pack index too?

Reply:
That can be tracked by the user but i could if needed

Reply:
Feels like pretty much any time you'd want the total packs, you'd also want the current index. I don't think adding the current index is a bad idea.

Reply:
done